GitHubReleases.ps1

```diff
@@ -196,5 +202,949 @@ function Get-GitHubRelease
+
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .PARAMETER OwnerName
+        Owner of the repository.
+        If not supplied here, the DefaultOwnerName configuration property value will be used.
+
+    .PARAMETER RepositoryName
+        Name of the repository.
+        If not supplied here, the DefaultRepositoryName configuration property value will be used.
+
+    .PARAMETER Uri
+        Uri for the repository.
+        The OwnerName and RepositoryName will be extracted from here instead of needing to provide
+        them individually.
+
+    .PARAMETER TagName
+        The name of the tag.
+
+    .PARAMETER Commitish
+        The commitsh value that determines where the Git tag is created from.
+        Cn be any branch or commit SHA.  Unused if the Git tag already exists.
+        Will default to the repository's default branch (usually 'master').
+
+    .PARAMETER Name
+        The name of the release.
+
+    .PARAMETER Body
+        Text describing the contents of the tag.
+
+    .PARAMETER Draft
+        Specifies if this should be a draft (unpublished) release or a published one.
+
+    .PARAMETER PreRelease
+        Indicates if this should be identified as a pre-release or as a full release.
+
+    .PARAMETER AccessToken
+        If provided, this will be used as the AccessToken for authentication with the
+        REST Api.  Otherwise, will attempt to use the configured value or will run unauthenticated.
+
+    .PARAMETER NoStatus
+        If this switch is specified, long-running commands will run on the main thread
+        with no commandline status update.  When not specified, those commands run in
+        the background, enabling the command prompt to provide status information.
+        If not supplied here, the DefaultNoStatus configuration property value will be used.
+
+    .EXAMPLE
+        New-GitHubRelease -OwnerName microsoft -RepositoryName PowerShellForGitHub -TagName 0.12.0
+
+    .NOTES
+        Requires push access to the repository.
+
+        This endpoind triggers notifications.  Creating content too quickly using this endpoint
+        may result in abuse rate limiting.
+#>
+    [CmdletBinding(
+        SupportsShouldProcess,
+        DefaultParameterSetName='Elements')]
+    [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSShouldProcess", "", Justification="Methods called within here make use of PSShouldProcess, and the switch is passed on to them inherently.")]
+    param(
+        [Parameter(ParameterSetName='Elements')]
+        [string] $OwnerName,
+
+        [Parameter(ParameterSetName='Elements')]
+        [string] $RepositoryName,
+
+        [Parameter(
+            Mandatory,
+            ValueFromPipelineByPropertyName,
+            ParameterSetName='Uri')]
+        [Alias('html_url')]
+        [string] $Uri,
+
+        [Parameter(Mandatory)]
+        [string] $TagName,
+
+        [string] $Commitish,
```

Comment:
Yeah, I need to think on this more.  It's a great point, because I really had no idea what `Committish` even meant when I authored this.  Let me reflect on how to make this more clear to users.

Reply:
Yep -- added aliases for `BranchName` and `Sha` (we use `Sha` elsewhere already, as opposed to `Commit`).